packages/vue/src/components/atoms/SfInput/SfInput.spec.js

```diff
@@ -1,31 +1,45 @@
 import { shallowMount } from "@vue/test-utils";
 import SfInput from "./SfInput.vue";
 describe("SfInput.vue", () => {
-  it("renders a input", () => {
-    const component = shallowMount(SfInput);
-    expect(component.classes("sf-input")).toBe(true);
+  it("renders component with defaults", () => {
+    const wrapper = shallowMount(SfInput);
+    expect(wrapper.classes("sf-input")).toBe(true);
   });
   it("renders label text when passed", () => {
     const label = "HelloWorld";
-    const component = shallowMount(SfInput, {
+    const wrapper = shallowMount(SfInput, {
       propsData: {
         label,
       },
     });
-    expect(component.find(".sf-input__label").text()).toBe(label);
+    expect(wrapper.find(".sf-input__label").text()).toBe(label);
   });
-  it("renders errorMessage slot with valid is false when passed", () => {
-    const errorMessage = "This field is required";
-    const component = shallowMount(SfInput, {
-      slots: {
-        "error-message": errorMessage,
+  it("emits input when triggered", () => {
+    const wrapper = shallowMount(SfInput, {
+      propsData: {
+        type: "password",
+        hasShowPassword: true,
+      },
+      listeners: {
+        input: () => {
+          expect(wrapper.vm.$data.input).toBeTruthy();
```

Comment:
There is an error about undefined here. What do you think about using `emitted()` instead? 
` expect(wrapper.emitted('input')).toBeTruthy()`
With tihs we also don't need `done()` :)